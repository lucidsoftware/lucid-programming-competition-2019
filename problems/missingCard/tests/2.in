15 4
6 13 7 1 14 3 4 2 11 10 15 12 9 5
10 3 1 12 6 15 8 11 7 13 14 4 2 5
12 3 5 4 8 11 9 6 10 13 15 1 2 14
3 5 7 12 10 8 13 14 9 6 4 2 15 11