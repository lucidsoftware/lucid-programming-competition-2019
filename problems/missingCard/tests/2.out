8
9
7
1